hummingbot/cli/utils/exchange_rate_conversion.py

```diff
@@ -28,20 +28,24 @@ def __init__(self):
         self.exchange_rate_config = {}
         self.exchange_rate = {}
         self.fetch_exchange_rate_task = None
+        # list of token required price adjustment
+        self.exchange_rate_conversion_list = []
         self.update_interval = 60.0
         self.started = False
         try:
-            exchange_rate_config = global_config_map["exchange_rate_conversion"].value
-            self.exchange_rate_config = {e[0]: {"default": e[1], "source": e[2]} for e in exchange_rate_config}
-            self.exchange_rate = {k: float(v.get("default", 1.0)) for k, v in self.exchange_rate_config.items()}
+            exchange_rate_conversion_config = global_config_map["exchange_rate_conversion"].value
+            exchange_rate_fetcher_config = global_config_map["exchange_rate_fetcher"].value
+            self.exchange_rate_conversion_list = {e[0] for e in exchange_rate_conversion_config}
```

Comment:
This makes no sense. First you declared `self.exchange_rate_conversion_list` as a list, but now you switch it to a set.